src/components/Switchs/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
Minor typo in root folder name. Rename it to `Switches` or just `Switch`.

Reply:
Hmm, good catch! It was original root folder name, but will change it